setup.py

```diff
@@ -1,14 +1,13 @@
 from setuptools import find_packages, setup
 
 setup(
-    name="ray_beam_runner",
+    name="ray_beam",
     packages=find_packages(where=".", include="ray_beam_runner*"),
     version="0.0.1",
     author="Ray Team",
     description="A Ray runner for Apache Beam",
     long_description="A distributed runner for Apache Beam built on top of "
     "distributed computing framework Ray.",
     url="https://github.com/ray-project/ray_beam_runner",
-    install_requires=[
-        "ray", "apache_beam"
-    ])
+    install_requires=["ray", "apache_beam"],
+)
```

Comment:
I think we should also target supporting the same Python versions as Ray:
```suggestion
      classifiers=[
          'Programming Language :: Python :: 3.6',
          'Programming Language :: Python :: 3.7',
          'Programming Language :: Python :: 3.8',
          'Programming Language :: Python :: 3.9',
      ],
)
```

Reply:
Added excepting 3.6 since it's deprecated.